src/main/scala/uk/gov/hmrc/play/http/HeaderCarrier.scala

```diff
@@ -114,10 +120,25 @@ object HeaderCarrier {
       headers.get(HeaderNames.trueClientPort),
       headers.get(HeaderNames.googleAnalyticTokenId),
       headers.get(HeaderNames.googleAnalyticUserId),
-      getDeviceId(cookies, headers)
+      getDeviceId(cookies, headers),
+      headers.get(HeaderNames.akamaiReputation).map(AkamaiReputation),
```

Comment:
I think we do still want to force Akamai-Reputation because we have downstream functionality that depends on it. By sneaking it in to the hard-coded headers with this change, we ensure that it could never be "accidentally" blacklisted by potential future configuration, which would break the downstream functionality.